src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyCatchBlockCheck.java

```diff
@@ -313,4 +313,14 @@ private static String getExceptionVariableName(DetailAST catchAst) {
         return variableName.getText();
     }
 
+    /**
+     * Returns the SLIST token corresponding to the catch block to be checked.
+     *
+     * @param ast a {@code DetailAST} value
+     * @return the SLIST token corresponding to the block to be checked
+     */
+    private static DetailAST findSlist(DetailAST ast) {
+        return ast.findFirstToken(TokenTypes.SLIST);
```

Comment:
please inline this code to `log(.....`, no need for method at all

Reply:
Oh okay, when you earlier mentioned to write a method, I thought it is a good practice. I'll change it right away